nights/2015-01-12-README.md

```diff
@@ -1,37 +1,35 @@
 # 2015-01-12
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Elias Holte
 
```

Comment:
Can you please add  link to the image gallery for this night here? An example is here: https://raw.githubusercontent.com/feder-observatory/processed_images/master/nights/2015-07-30-README.md